package.js

```diff
@@ -1,6 +1,6 @@
 Package.describe({
   name: 'dispatch:run-as-user',
-  version: '0.0.2',
+  version: '1.0.0',
```

Comment:
marked as a major since the behaviour of `Meteor.runAsUser` is replaced by `Meteor.runAsRestrictedUser`